tests/manual/placeholder.html

```diff
@@ -1 +1,5 @@
+<style type="text/css">
+	.myUi { display: inline-block; width: 6px; height: 6px; position: relative; top: -16px; background: #AA0000; pointer-events: none; }
```

Comment:
I don't think we should use `display: inline-block` since we know there are problems with it. Why not `display: inline;`?